README.md

```diff
@@ -295,6 +295,15 @@ The status of tracking requests can be observed through emitted events. `message
 constructorio.tracker.on(messageType, callback);
 ```
 
+#### Dispatched events
+The `search`, `browse` and `recommendations` modules may dispatch custom events on `window` with response data when a request has been completed. The event name follows the following structure: `ConstructorIO.[moduleName].[methodName].response`. Example consuming code can be found below:
```

Comment:
Shall we start working towards `cio.client.search.getSearchResults.complete`, or something to that effect?